test/src/api/expect/testExpectVisible.js

```diff
@@ -359,64 +359,64 @@ describe('expect.visible', function() {
 
       let expect = this.client.api.expect.element('#weblogin').to.be.visible;
 
-      return this.client.start(function() {
-        assert.ok(expect.assertion.retries >= 2);
+      return this.client.start(function () {
+        strictEqual(expect.assertion.retries, 2);
```

Comment:
why is this change necessary? Wouldn't it create flakiness?

Reply:
I can't remember... will revert the change